File: gin/entity/po/like.go
package po

import "time"

type Like struct {
	ID        uint      `gorm:"column:id" json:"id"`
	ArticleID uint      `gorm:"column:article_id" json:"articleId"`
	UserID    uint      `gorm:"column:user_id" json:"userId"`
	CreatedAt time.Time `gorm:"column:created_time" json:"createdAt"`
}

func (Like) TableName() string {
	return "likes"
}

File: fastapi/app/internal/models/like.py
from datetime import datetime

from app.core.db import Base
from sqlalchemy import Column, DateTime, Integer


class Like(Base):
    """点赞实体类"""

    __tablename__ = "likes"
    id = Column(Integer, primary_key=True, index=True, autoincrement=True)
    article_id = Column(Integer, nullable=False, index=True)
    user_id = Column(Integer, nullable=False, index=True)
    created_time = Column(DateTime, nullable=True, default=datetime.utcnow)
